zoe-cli/test/com/adevinta/oss/zoe/cli/helpers.kt

```diff
@@ -72,17 +76,28 @@ suspend fun ExpectScope.zoe(
     return res
 }
 
-val testConfDir = File("testResources/env")
+private val testModule = module {
+    single { customizeEnvConfig }
+}
 
-fun testDockerCompose() =
-    DockerCompose(testConfDir.resolve("docker-compose.yml")).apply {
-        waitingFor("broker", Wait.forListeningPort())
-        waitingFor("schema-registry", Wait.forHttp("/subjects").forStatusCode(200))
-    }
+class TestcontainersListener : TestListener {
```

Comment:
Now that we have a `testcontainers.kt` you can move this listener there :)

Reply:
Good point 😅!